WeakAuras/Types.lua

```diff
@@ -524,6 +524,7 @@ Private.format_types = {
             return ""
           end
           local name, realm = WeakAuras.UnitNameWithRealm(unit)
+          name = WeakAuras.GetName(name)
```

Comment:
this won't work as chars in lib are saved in format Name-Realm you can either build the string before request or use another overwrite for  Lib.UnitFullName(unit) intead of UnitFullName e.g. in  WeakAuras.UnitNameWithRealm(unit) WeakAuras.lua line 5713ff

Reply:
WeakAuras.UnitNameWithRealm can't return a fantasy value as it's used in triggers